tests/test_plugin/Model/Table/HelloWorld101010ArticlesTable.php

```diff
@@ -35,15 +35,10 @@ public function clearArticle($article)
     }
 
     /**
-     * @return \Cake\Datasource\EntityInterface
+     * @return \Cake\Datasource\EntityInterface|array<\Cake\Datasource\EntityInterface>
```

Comment:
How does this return an array of entities?

Reply:
https://github.com/cakephp/cakephp/blob/4.x/src/Datasource/QueryTrait.php#L501

It didn't work with just defining
```
@return \Cake\Datasource\EntityInterface|array
```

Reply:
But you misunderstand the array, it means `array<string, mixed>` never multiple entities!

Reply:
indeed, I didn't know that. let me fix that.